E621Client/Area/Iqdb/E621Client.cs

```diff
@@ -21,44 +21,17 @@ public partial class E621Client
         // an IQDB query seems to prevent 429 responses.
         private const int IqdbRequestInterval = 0;
         private const int IqdbRequestDelay = 2000;
+        
+        /// <inheritdoc cref="IE621Client.IqdbAllowedFormats"/>
```

Comment:
It seems like you forgot to point the references to the constants to the new `E621Constants` class on multiple occasions. This should be `E621Constants.IqdbAllowedFormats`.

Reply:
True but they are actually not needed anymore because it these properties were static and have now been moved to the `E621Constants` class.